Tasks/PublishToPyPI/README.md

```diff
@@ -1,4 +1,4 @@
-# PyPI Publisher
+# Publish to PyPI
```

Comment:
I think the task name should stay as is.  There are plenty of other tasks named with nouns, and I think it's helpful to have "PyPI" in first place.